geps/gep-2162.md

```diff
@@ -177,12 +177,12 @@ Once the GatewayClass features support are is published into the status we could
 
 2. Build tooling to check and warn when unsupported features are used.
 
-3. Add a table with feature name and description to document what each feature means.
-
 ### Add Gateway API Version field to the GatewayClass Status
 
 We got some feedback that it will be useful to indicate what what Gateway API version the implementation supports. So when we have supported features published in the GatewayClass Status, users will also be able to understand that those are the supported features for a specific Gateway API version.
 
 This work is likely to require its own small GEP but ideally what this field would mean is that an implementation supports Max(vX.X). 
 
 The value of it is to provide a better user experience and also more foundation for tools to be able to warn for example when a GatewayClass and CRDs have mismatched versions.
+
+### Add a table with feature name and description to document what each feature means
```

Comment:
Assuming you meant to add some related content below this heading?

Reply:
Honestly I think it's also fine to just leave it in future work. But to complete this I think we'd want to document how this feature works + what each entry in the list means. So maybe that would live somewhere like https://gateway-api.sigs.k8s.io/api-types/gatewayclass/. With that said, I think a follow up is fine for that. It just feels awfully funny to have a heading with no content below it as the case is now.

Reply:
added a line to describe the work. 
Hopefully this addresses the concern